lib/elixir_tic_tac_toe_basic/cli.ex

```diff
@@ -7,17 +7,10 @@ defmodule ElixirTicTacToeBasic.CLI do
     ElixirTicTacToeBasic.start(%{
       ui: ElixirTicTacToeBasic.UI,
       presenter: ElixirTicTacToeBasic.Presenter,
-      board: %{
-        {0, 0} => 1,
-        {0, 1} => 2,
-        {0, 2} => 3,
-        {1, 0} => 4,
-        {1, 1} => 5,
-        {1, 2} => 6,
-        {2, 0} => 7,
-        {2, 1} => 8,
-        {2, 2} => 9
-      }
+      player: ElixirTicTacToeBasic.Player,
+      board: Helpers.EmptyBoard.empty_board(),
```

Comment:
**praise**: I like the way you encapsulated the board construction!

**suggestion (non-blocker)**: Avoid "helpers", consider `Board.new`

"Helpers" modules or classes often become a dumping ground for code that doesn't live together, however, I like that you included an additional `EmptyBoard` namespace to help mitigate that.

I think it is conventional in elixir to offer a "construction" function as `.new`.

```elixir
board: Board.new
```